OrchestrationAirflow/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "google/cloud-orchestration-airflow",
+    "description": "Google Cloud Orchestration Airflow Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "autoload": {
+        "psr-4": {
+            "Google\\Cloud\\Orchestration\\Airflow\\Service\\": "src",
```

Comment:
Why do these use the namespace `\\Service` at the end? This is not consistent with all the other libraries we have (they do not include an additional namespace after the package name)

I am guessing that the [protos](https://github.com/googleapis/googleapis/blob/master/google/cloud/orchestration/airflow/service/v1/environments.proto) use service but we chose to not include it in our package name. If that's the case, then I agree with the decision!

Reply:
It's mimicking the proto directory structure, odd choice by the team, I agree.